<launch>
  
  <arg name="agent_node_param"  default="$(find-pkg-share agent_node)/config/agent_node.param.yaml"/>
  <!-- cnn node -->
  <node pkg="agent_node" 
        exec="agent_node"
        name="agent_node"
        output="screen">

    <!-- パラメータ -->
    <param from="$(var agent_node_param)"/>
    <!-- トピックのremap -->
    <remap from="/scan" to="/scan_filtered"/>
    <remap from="/cmd_drive" to="/ackermann_cmd"/>
  </node>


</launch>